<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeuroWidget</class>
 <widget class="QWidget" name="NeuroWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbl_sample">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Samples Captured: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Status: none</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_train">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Train Neural Nework</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_loadnn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Neural to LUT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Training data</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
